### PR TITLE
bump `@guardian/libs` to `10.0.0`

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

bump `@guardian/libs` to `10.0.0`

## Why?

to keep everything up to date, and align peerdependencies

### CODE REVIEW

File: dotcom-rendering/package.json [hunk 1]
```diff
@@ -225,6 +225,7 @@
 		"ts-jest": "^24.3.0",
 		"ts-loader": "^9.3.0",
 		"ts-unused-exports": "^8.0.0",
+		"tslib": "^2.4.0",
```
Comment (mxdvl) [inline dotcom-rendering/package.json:228]: Why has this come back?

**Review by mxdvl — APPROVED** [review]
Looks fine from a `dotcom-rendering` perspective

**Review by JamieB-gu — COMMENTED** [review]
Looks good, just one comment about the `package.json` changes.

File: apps-rendering/package.json [hunk 2]
```diff
@@ -1,127 +1,128 @@
...
+		"react-dom": "^17.0.2",
+		"react-test-renderer": "^17.0.2",
+		"regenerator-runtime": "^0.13.9",
+		"require-from-string": "^2.0.2",
+		"source-map-support": "^0.5.21",
+		"thrift": "^0.16.0",
+		"ts-jest": "^28.0.7",
+		"ts-loader": "^9.4.1",
+		"ts-node": "^10.8.2",
+		"tslib": "^2.4.0",
```
Comment (JamieB-gu) [inline apps-rendering/package.json:118]: Similar comment to @mxdvl, curious what this is used for?

Comment (joecowton1) [reply apps-rendering/package.json:118]: its a peer dep of libs 10

**Pull request merged**